README.md

```diff
@@ -118,6 +119,32 @@ Additional notes for non-coders
    the same code spot-check process a new contributor needs to undergo at the
    time they add the code.
 
+Backdrop Bug Squad
+------------------
+
+The Backdrop Bug Squad consistes of members of the backdrop contrib group who
+will help other contributors stay on top of minor bug fixes and UI improvements.
+
+Responsibilities include:
+- Communicating with existing maintainers.
+- Reviewing Pull Requests for Squad-Merge eligibility.
+- Merging eleigable Pull Requests.
+- Determining if a project has been abandoned.
+
+Fixes eligible for Squad-Merge inlude:
+- Minor Bugs (PHP notices, validation errors, missing == or ===, etc).
+- Minor User Interface changes (typos, spelling, grammar, terminology, etc).
+- Any change that could be accepted after a "feature freeze".
```

Comment:
I would add "any packaging issue preventing the project from being downloaded and installed via the Project Browser". I technically was a bad boy and didn't follow the correct protocol for fixing a packaging release, no code changes, but since users couldn't install the current release, I thought it was a more urgent matter.

Reply:
good one, adding.